xml/System.Net.Http.Metrics/HttpMetricsEnrichmentContext.xml

```diff
@@ -24,8 +24,11 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides functionality for enriching the `http.client.request.duration` metric.</summary>
+    <remarks>Enrichment is done on per-request basis by callbacks registered with <see cref="M:System.Net.Http.Metrics.HttpMetricsEnrichmentContext.AddCallback(System.Net.Http.HttpRequestMessage,System.Action{System.Net.Http.Metrics.HttpMetricsEnrichmentContext})" />.
+ The callbacks are responsible for adding custom tags via <see cref="M:System.Net.Http.Metrics.HttpMetricsEnrichmentContext.AddCustomTag(System.String,System.Object)" /> for which they can use the request, response and error information exposed on the <see cref="T:System.Net.Http.Metrics.HttpMetricsEnrichmentContext" /> instance.
+ 
+ &gt; [!IMPORTANT] &gt; The <see cref="T:System.Net.Http.Metrics.HttpMetricsEnrichmentContext" /> intance must not be used from outside of the enrichment callbacks.</remarks>
```

Comment:
@gewarren do you know how to represent an `[!IMPORTANT]` tag with plain xml? If there's no easy way, then we need to convert these remarks to markdown so that we can show the `[!IMPORTANT]` header with the proper formatting:

```suggestion
    <remarks>
      <format type="text/markdown"><![CDATA[

## Remarks

Enrichment is done on per-request basis by callbacks registered with <xref:System.Net.Http.Metrics.HttpMetricsEnrichmentContext.AddCallback(System.Net.Http.HttpRequestMessage,System.Action{System.Net.Http.Metrics.HttpMetricsEnrichmentContext})>.

The callbacks are responsible for adding custom tags via <xref:System.Net.Http.Metrics.HttpMetricsEnrichmentContext.AddCustomTag(System.String,System.Object)> for which they can use the request, response and error information exposed on the <xref:System.Net.Http.Metrics.HttpMetricsEnrichmentContext> instance.
 
 > [!IMPORTANT]
 > The <xref:System.Net.Http.Metrics.HttpMetricsEnrichmentContext> instance must not be used from outside of the enrichment callbacks.
 
      ]]></format>
    </remarks>
```